packages/office-ui-fabric-react/src/components/Panel/Panel.styles.ts

```diff
@@ -119,33 +208,16 @@ export const getStyles = (props: IPanelStyleProps): IPanelStyles => {
         right: 0,
         bottom: 0
       },
-      !isOpen &&
-        !isAnimating &&
-        isHiddenOnDismiss && {
-          visibility: 'hidden'
-        },
```

Comment:
Unnecessary style as there is already a `hiddenPanel` mergeStyles class applied to a parent element of the `root` area doing the exact same thing on line 222.